lib/answers.ts

```diff
@@ -17,64 +17,23 @@ export const getAnswer = (answers: Answer[], entity, variable?, id?) => {
   return answer ? answer.value : undefined
 }
 
-export const getAnswerIndex = (
-  answers: Answer[],
-  entityName,
-  id,
-  fieldName
-) => {
-  return answers.findIndex((answer) => {
-    if (entityName && id && fieldName) {
-      return (
-        answer.entityName === entityName &&
-        answer.id === id &&
-        answer.fieldName === fieldName
-      )
-    } else if (entityName && id) {
-      return answer.entityName === entityName && answer.id === id
-    } else if (entityName) {
-      return answer.entityName === entityName
-    }
-  })
-}
-
 // Nécessaire si la question est optionnelle
 export function nullifyUndefinedValue(value) {
   return value === undefined ? null : value
 }
 
-export function storeAnswer(answers, newAnswer, enfants?) {
+export function storeAnswer(answers, newAnswer) {
```

Comment:
On pourrait ajouter du typage dans les paramètres ici ? 
(Dis moi si c'est superflu ?)
```suggestion
export function storeAnswer(answers: Answer[], newAnswer: Answer) {
```

Reply:
Je vais regarder, si les types sont bien fait ça devrait pas poser de soucis, sinon je regarderais pour les corriger.